signals/parser/schema.py

```diff
@@ -28,6 +29,25 @@ def create_objects(self, object_json):
             data_object = DataObject(object_name, data_object_json[object_name])
             self.data_objects[data_object.name] = data_object
 
+    # Note: Why do we call them "apis" in method names, but then call them "urls" in the schema/code
```

Comment:
A single API is called an endpoint here - a URL is a collection of endpoints: ex. we have /api/posts/ which could have a GetAPI, PostAPI, and DeleteAPI

Agree we could have some better consistent naming